app/models/grade_entry_form.rb

```diff
@@ -22,6 +22,8 @@ class GradeEntryForm < ActiveRecord::Base
   validates_presence_of     :short_identifier
   validates_uniqueness_of   :short_identifier, case_sensitive: true
 
+  validates_inclusion_of    :is_hidden, in: [true, false]
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer the new style validations `validates :column, inclusion: value` over `validates_inclusion_of`.